Documentation/slip-rate-on-DR.rst

```diff
@@ -0,0 +1,17 @@
+Slip-rate imposed on a DR boundary
+===================================
+
+This "pseudo" friction law allows imposing slip-rate on a dynamic rupture boundary.
+The FL id for this friction law is 33.
+The advantage of this approach compared to a multi point-source representation is that the fault slip is not condensed to points. 
+Therefore the discontinuity of the displacement across the fault can be accurately accounted for.
+In addition, no spurious waves, related to the multi point-sources representation, are generated.
+
+The current implementation allows imposing kinematic models parameterized by regularized Yoffe function (see Tinti et al., 2005, doi:10.1029/2005JB003644).
```

Comment:
Is the formula for a Yoffe function very long? If not it would be useful to include it here.

Reply:
yes, very long:
![tinti1](https://user-images.githubusercontent.com/13536910/133977627-bd8686d3-6248-4496-99a3-8307bfca7399.png)
![tinti2](https://user-images.githubusercontent.com/13536910/133977631-94bcf6b0-2847-4301-9dcf-cf1c3b9b0b34.png)

Reply:
How about a picture then?

Reply:
ok

Reply:
Shouldn't you add 10.1785/0120040177 to the docs, too?

Reply:
good point, I was pointing to the wrong paper.